### PR TITLE
Developer Tools week 3 day 5 update

### PR DESCRIPTION
**Pull request by jgw96**
---

⚠️ Merge/Release date 3/4

---

### CODE REVIEW

File: docs/30DaysOfPWA/dev-tools/02.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # 3.2: Get-started Tools: Converting your web app to a PWA!
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/02.md:1]: This PR is about day 5, why was day 2 also changed in this PR?

Comment (captainbrosset) [reply docs/30DaysOfPWA/dev-tools/02.md:1]: Ah it looks like you might just need to merge from `main` once day 2 gets there.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 2]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
 
 ---
+
+Hello! For today’s post on the 30 Days of PWA series, we will be learning how to audit our PWA using various tools to help improve our PWA for our users. 
+
+## Why audit your PWA
+
+So, why should we audit our PWA anyways? And what does this even mean?
+
+When we are talking about auditing your PWA, we mean using tools to ensure that your PWA is:
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:30]: A little before, we use "we will be learning" and "our PWA". But now we say "your PWA". Let's try to choose one way to talk to the reader, and use it consistently throughout. Personally, I like using "we" as you did above.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 3]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+## Why audit your PWA
+
+So, why should we audit our PWA anyways? And what does this even mean?
+
+When we are talking about auditing your PWA, we mean using tools to ensure that your PWA is: 
+
+- Installable from the browser
+- Ready to submit to app stores such as the Microsoft Store
+- Accessibile
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:34]:
```suggestion
- Accessible
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 4]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+- Fast
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:35]: I would add another bullet point here:

- Follows cross-browser development best practices

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 5]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+and more. We do this to, in an automated way, ensure our PWA is the best it can be for users. Users want high quality apps that are accessible, fast and discoverable, and auditing ensures we are meeting these important wants.
+
+Today we are going to use PWABuider, Webhint and Edge Devtools to audit our PWA, let's dive in!
+
+## Audit with PWABuilder
+
+Let's start with PWABuilder. The focus of the Audit's PWABuilder does is to ensure your PWA is, high-quality, installable and ready to publish to app stores like the Microsoft Store. To do this, PWABuilder looks at your
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:43]:
```suggestion
Let's start with PWABuilder. The focus of the audit PWABuilder does is to ensure your PWA is high-quality, installable, and ready to publish to app stores like the Microsoft Store. To do this, PWABuilder looks at the:
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 6]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+and audits them based on a set of requirements. 
+These also happen to be the technical essentials you need for your app to be considered a PWA. 
+
+| Web Manifest | Service Worker | Security
+| ------- | ----------- | ----------- |
+| Properly linked + registered | Properly linked + registered | Uses HTTPS
+| Contains required fields for installation | Works Offline | Valid HTTPS Setup
+| Icons follow best practices | Suggested features to improve your offline experience | No HTTP content on page
+| Suggested fields to improve your PWA or enable new features | | |
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:57]: I think it's more natural to read a table row by row. So I would reverse this table so that the manifest, sw, and security headers are actually row headers. And you can each row entirely before jumping to the next topic.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 7]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+Each section has a set of Required audits. These are audits that are required to be in the app stores and for installation from the browser. There are also Recommended and Optional audits for each section. These audit things such as, for example, manifest fields you can add to improve the installation experience of your PWA, but are not required.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:59]:
```suggestion
Each of the section above has a set of required audits in PWABuilder. These are audits that are required for your PWA to be in the app stores and for installation from the browser. There are also recommended and optional audits for each section, for example, manifest members you can add to improve the installation experience of your PWA.
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 8]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+To get started with PWABuilder's audits, you can use either https://pwabuilder.com or the [PWA Studio VSCode Extension](https://marketplace.visualstudio.com/items?itemName=PWABuilder.pwa-studio).
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:61]: You might want to create a link for pwabuilder.com instead of just using an inline URL. This way we can create an aka.ms link for it too.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 9]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+### Getting our audits from pwabuilder.com
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:63]: Here is another example of mixing "our" (here in the title) and "your" (below in the bullet list).
I think we should settle on using "we"/"our" throughout, but you can probably remove it from the section headers. I think section headers are more natural when neutral, like this "Getting audits from pwabuilder.com".
And the next one: "Getting audits from PWA Studio".

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 10]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+Already have a published and live PWA that you want to audit? 
+
+- Grab your URL and head over to https://pwabuilder.com
+- Enter the URL to your PWA on the homepage of pwabuilder.com and click Start
+![We have entered our URL into the input on the homepage of PWABuilder](_media/05/enter-url.png)
+- pwabuilder.com will now analyze your PWA and return your audits
+![A screenshot of the report-card page in PWABuilder showing our audits](_media/05/report-card.jpeg)
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:70]: Can we add a short section about what the scores are? How to read them?

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 11]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+### Getting our audits with PWA Studio
+PWA Studio enables you to use PWABuilders audits while your building your PWA, not just after your PWA has gone live.
+
+- Install PWA Studio from the [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=PWABuilder.pwa-studio)
+- Open your codebase in VSCode
+- Click the PWABuilder icon on the left side of VSCode
+![The PWABuilder icon on the left side of VSCode](_media/05/icon-on-left.png)
+- You will now see your audits
+
+![The audits panels in PWA Studio](_media/05/pwa-studio.png)
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:81]: It might be good to say a few words about how the extension doesn't use scores, unlike pwabuilder.com. It's not immediately obvious to me how to read the audit in the screenshot, so a few words about this would help (also for people relying on assistive technology).

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 12]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+## Audit accessibility with Webhint
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:83]: Webhint doesn't only audit accessibility. It has a lot of different hints in various categories: browser compat, performance, security, accessibility, pitfalls, and even PWA.
So I would suggest renaming this header to something like "Audit accessibility, speed, compatibility, and more with Webhint in Edge DevTools".

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 13]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+Now on to auditing accessibliity. We want to ensure our apps are inclusive and built for everyone and anyone.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:85]: Based on my previous comment, the first sentence here should be changed, or maybe just dropped.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 14]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+- You will now see your audits
+
+![The audits panels in PWA Studio](_media/05/pwa-studio.png)
+
+## Audit accessibility with Webhint
+
+Now on to auditing accessibliity. We want to ensure our apps are inclusive and built for everyone and anyone. 
+You may have heard of [Webhint](https://webhint.io/) before, but did you know it's audits are built into the [Issues panel in Edge devtools?](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/issues/)
+
+The Issues panel allows you to quickly see any accessibility issues your app may have. But, it does not only tell you what is wrong, but also exactly which HTML element the problem affects and how to fix it.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:88]:
```suggestion
The Issues panel allows you to quickly see any of the issues webhint detected in your app. On top of this, it also tells you which HTML element the problem affects and how to fix it.
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 15]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+![The issues panel in Edge devtools showing an accessibility issue](_media/05/webhint.png).
+
+To use it:
+- Open your app in Edge
+- Open [Edge devtools](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/)
+- Refresh your app, because some issues are reported based on network requests. Notice the updated count in the Issues counter.
+- Select the Issues tab. The Issues panel opens with issues grouped into different categories.
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:96]: I would reverse the order of these last 2 bullet points. So: 1. open edge 2. open devtools 3. open issues panel and check for issues and 4. refresh the app and check that there are more issues which depend on page load.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 16]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+## Audit performance with Lighthouse and Edge Devtools
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:98]: I personally wouldn't necessarily restrict this to Edge. If you rephrase this to "Audit performance with Lighthouse" and then mention in the section below that this works both in Chrome and Edge, that makes it more inclusive to everyone.
Right now, it's a bit like saying that lighthouse only exists in Edge.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 17]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+**Note** Edge devtools can also help you evaluate and improve your apps runtime performance. Check our [docs](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/) to learn more.
+
+Edge devtools has the [Lighhouse](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/speed/get-started) tool built in. Lighthouse can quickly evaluate our apps loading performance and give us suggestions on how to make it load faster. It will evaluate our app running on our device, but with CPU throttling and Network throttling to better match what you users experience. When talking about loading performance, it is important to remember that we need to ensure our apps load fast on our users devices, not just our development machines. Lighthouse enables us to do this directly from our development device.
+
+To get started auditing your loading performance with Lighthouse:
+
+- Open your app in Edge
+- Open [Edge devtools](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/)
+- Open the Lighthouse tab in devtools ![A screenshot of the lighthouse tab at the top of devtools](_media/05/lighthouse.png)
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:110]: You should move the image to be in its own block. Not inline with the bullet point.

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 18]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+- Click `Generate Report` to get a baseline report
+- Analyze the report and make the suggested changes to your app.
+
+For a full walkthrough on using Lighthouse to improve the loading performance of your application, check our [docs](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/speed/get-started). 
+
+
+Using these tools, we can quickly make changes to our PWA to ensure it provides the best experience possible for our users. And, because PWAs instantly updateable just like a normal website, we can get these changes out to our users quickly!
```
Comment (captainbrosset) [inline docs/30DaysOfPWA/dev-tools/05.md:117]:
```suggestion
Using these tools, we can quickly make changes to our PWA to ensure it provides the best experience possible for our users. And, because PWAs are instantly updateable like normal websites, we can get these changes out to our users quickly!
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]
Great content. I made some comments below.

File: docs/30DaysOfPWA/dev-tools/02.md (hunk 1 above)
Comment (beth-panx) [reply docs/30DaysOfPWA/dev-tools/02.md:1]: @jgw96 can you manage this merge on your end?

Comment (beth-panx) [reply docs/30DaysOfPWA/dev-tools/02.md]: NVM I did it :)

File: docs/30DaysOfPWA/dev-tools/05.md (hunk 17 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/dev-tools/05.md:110]:
```suggestion
- Open the Lighthouse tab in devtools 

![A screenshot of the lighthouse tab at the top of devtools](_media/05/lighthouse.png)

```

File: docs/30DaysOfPWA/dev-tools/05.md (hunk 2 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/dev-tools/05.md:30]: I also like we. I need to revisit my articles because I'm pretty certain I switch back and forth too 😅

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 19]
```diff
@@ -12,10 +14,113 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+
+To use it:
+- Open your app in Edge
+- Open [Edge devtools](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/)
+- Refresh your app, because some issues are reported based on network requests. Notice the updated count in the Issues counter.
+- Select the Issues tab. The Issues panel opens with issues grouped into different categories.
+
+## Audit performance with Lighthouse and Edge Devtools
+
+And we are now ready to audit our apps performance. When I say performance here, I mean loading Performance or "How fast your PWA loads on a certain device with a certain network connection".
```
Comment (zateutsch) [inline docs/30DaysOfPWA/dev-tools/05.md:100]:
```suggestion
We are now ready to audit our app's loading performance, or how fast your PWA loads on different devices with varying network connections.
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 20]
```diff
@@ -1,6 +1,8 @@
 # 3.5: Auditing tools
 
-## *Coming soon!*
+**Author: Justin Willis (Software Engineer on the PWABuilder team) [@justinwillis96](https://twitter.com/Justinwillis96)**
+
+Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this project? Check out our [Kickoff](../kickoff.md) post to get more details on the content roadmap and contributors. Now, let's dive in!
 
 **Author: Justin Willis [@justinwillis96](https://twitter.com/Justinwillis96)**
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:7]:
```suggestion
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 21]
```diff
@@ -1,6 +1,8 @@
 # 3.5: Auditing tools
 
-## *Coming soon!*
+**Author: Justin Willis (Software Engineer on the PWABuilder team) [@justinwillis96](https://twitter.com/Justinwillis96)**
+
+Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this project? Check out our [Kickoff](../kickoff.md) post to get more details on the content roadmap and contributors. Now, let's dive in!
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:5]:
```suggestion
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 22]
```diff
@@ -65,17 +65,32 @@ Already have a published and live PWA that you want to audit?
 - pwabuilder.com will now analyze our PWA and return our audits
 ![A screenshot of the report-card page in PWABuilder showing our audits](_media/05/report-card.jpeg)
 
+Each section is given an overall score. If we click on that section, it will give us more details on our score, including what you can add or change to get a better score. Towards the bottom of the page, we can see our total score.
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:68]:
```suggestion
Each section is given an overall score. If we click on that section, it will give us more details on our score, including what we can add or change to get a better score. Towards the bottom of the page, we can see our total score.
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 23]
```diff
@@ -65,17 +65,32 @@ Already have a published and live PWA that you want to audit?
...
+![A screenshot showing the total score located towards the bottom of the report page](_media/05/total-score.png)
+
+If our total score is green, then our PWA is ready to publish and we are ready to click next to start packaging our PWA.
+
+If our total score is yellow this means that our PWA is most likely ready to publish, but may be missing out on Recommended Features such as having a background_color in your Web Manifest. The section that is lowering our score will also have a yellow score. At that point, we should click on it to see what we need to add.
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:73]:
```suggestion
If our total score is yellow this means that our PWA is most likely ready to publish, but may be missing out on Recommended Features such as having a `background_color` in our Web Manifest. The section that is lowering our score will also have a yellow score. At that point, we should click on it to see what we need to add.
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 24]
```diff
@@ -65,17 +65,32 @@ Already have a published and live PWA that you want to audit?
...
+
+If our score is red, this means our PWA is not ready to publish to the web or app stores and needs additional work. The section with the problem will also have a red score. As above, you can click on that section to see what needs to be added next.
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:75]:
```suggestion
If our score is red, this means our PWA is not ready to publish to the web or app stores and needs additional work. The section with the problem will also have a red score. As above, we can click on that section to see what needs to be added next.
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 25]
```diff
@@ -65,17 +65,32 @@ Already have a published and live PWA that you want to audit?
...
+
 ### Getting our audits with PWA Studio
 PWA Studio enables you to audit with PWABuilder during development, not just after our PWA has gone live.
 
 - Install PWA Studio from the [VSCode Marketplace](https://aka.ms/learn-PWA/30Days-3.5/pwa-studio)
 - Open our codebase in VSCode
 - Click the PWABuilder icon on the left side of VSCode
 ![The PWABuilder icon on the left side of VSCode](_media/05/icon-on-left.png)
-- You will now see your audits
+- You will now see your audits.
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:84]:
```suggestion
- We will now see our audits
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 26]
```diff
@@ -65,17 +65,32 @@ Already have a published and live PWA that you want to audit?
...
+![A screenshot showing the total score located towards the bottom of the report page](_media/05/total-score.png)
+
+If our total score is green, then our PWA is ready to publish and we are ready to click next to start packaging our PWA.
+
+If our total score is yellow this means that our PWA is most likely ready to publish, but may be missing out on Recommended Features such as having a background_color in your Web Manifest. The section that is lowering our score will also have a yellow score. At that point, we should click on it to see what we need to add.
+
+If our score is red, this means our PWA is not ready to publish to the web or app stores and needs additional work. The section with the problem will also have a red score. As above, you can click on that section to see what needs to be added next.
+
 ### Getting our audits with PWA Studio
 PWA Studio enables you to audit with PWABuilder during development, not just after our PWA has gone live.
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:78]:
```suggestion
PWA Studio enables us to audit with PWABuilder during development, not just after our PWA has gone live.
```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 27]
```diff
@@ -65,17 +65,32 @@ Already have a published and live PWA that you want to audit?
...
+- You will now see your audits.
 
 ![The audits panels in PWA Studio](_media/05/pwa-studio.png)
 
+For PWA Studio, we are using a simpler way of showing our audits. As it shows in the above screenshot, there are three sections again, but slightly different this time: Web Manifest, Service Worker, Store Ready.
+
+For each section, you will see a list of items. If that item is good to go, it will have a checkmark next to it, if not, it will have a caution symbol. For every item, you can click on the info icon in that item to learn more about it and how to fix the audit.
+
+For the best PWA, we recommend checking off as many items as you can. If you are missing something that is required, PWA Studio will give you an alert when trying to publish to the web or package your PWA for the app stores.
+
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:93]:
```suggestion
PWA Studio uses a simpler way of showing audits. As it shows in the above screenshot, there are three sections again, but slightly different this time: Web Manifest, Service Worker, Store Ready.

For each section, you will see a list of items. If that item is good to go, it will have a checkmark next to it, if not, it will have a caution symbol. For every item, you can click on the info icon in that item to learn more about it and how to fix the audit.

For the best PWA, we recommend checking off as many items as you can. If you are missing something that is required, PWA Studio will give you an alert when trying to publish to the web or package your PWA for the app stores.

```

File: docs/30DaysOfPWA/dev-tools/05.md [hunk 28]
```diff
@@ -12,10 +10,129 @@ Welcome to **Week 3 Day 5** of #30DaysOfPWA! Want to learn more about this proje
...
+## Why audit a PWA
+
+So, why should we audit our PWA anyways? And what does this even mean?
+
+When we are talking about auditing our PWA, we mean using tools to ensure that our PWA is: 
+
+- Installable from the browser
+- Ready to submit to app stores such as the Microsoft Store
+- Accessible
+- Fast
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/05.md:31]:
```suggestion
- Fast
- Follows cross-browser development best practices
```

**Review by beth-panx — APPROVED** [review]

File: docs/30DaysOfPWA/dev-tools/07.md [hunk 29]
```diff
@@ -1,7 +1,5 @@
 # 3.7: Packaging tools
 
-## *Coming soon!*
-
 **Author: Jaylyn Barbee [@jaylynsatwork](https://twitter.com/jaylynsatwork)**
```
Comment (beth-panx) [inline docs/30DaysOfPWA/dev-tools/07.md:3]:
```suggestion
## *Coming soon!*

**Author: Jaylyn Barbee [@jaylynsatwork](https://twitter.com/jaylynsatwork)**
```

**Comment by beth-panx** [comment]
@zateutsch @captainbrosset I worked with Justin to update everything you mentioned. And did a final pass. Going to merge this for tomorrow.
